tfbpshiny/utils/format_number_for_display.py

```diff
@@ -0,0 +1,44 @@
+import collections.abc
+
+
+def format_to_sci_notation(data_input):
+    """
+    Formats a number or an iterable of numbers into scientific notation with two decimal
+    places.
+
+    Args:
```

Comment:
I didn't notice this earlier -- make sure to match the docstring format for the rest of the repo. Use sphinx formating. You will need to make sure the AI outputs this correctly